google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java

```diff
@@ -205,7 +207,8 @@ public void cancelled(Context context) {
         logger.log(Level.FINE, "Retryable exception, will sleep and retry", e);
         backoffSleep(context, backOff);
       } catch (Exception e) {
-        throw Throwables.propagate(e);
+        Throwables.throwIfUnchecked(e);
+        throw new RuntimeException(e);
```

Comment:
I agree with codacy-bot (even though the replaced Throwables.propagate() is doing exactly same. Can the exception be a non-retriable SpannerException instead of being raw RuntimeException?